src/API/Reports/Orders/DataStore.php

```diff
@@ -325,6 +325,7 @@ protected function normalize_order_by( $order_by ) {
 	 */
 	protected function include_extended_info( &$orders_data, $query_args ) {
 		$mapped_orders    = $this->map_array_by_key( $orders_data, 'order_id' );
+		$related_orders   = $this->get_orders_with_parent_id( $mapped_orders );
 		$products         = $this->get_products_by_order_ids( array_keys( $mapped_orders ) );
```

Comment:
@octaedro I think we also want to pass in the `$related_orders` id's into the `get_products_by_order_ids`.
Probably just combine `array_keys( $mapped_orders )` and `array_keys( $related_orders )`.

Currently the related order products only show if the original order is part of the date range, if not it won't show the products. This might likely happen if someone ordered it a week ago and decided to return, see screenshots:

List without the original order (see no products):
<img width="1661" alt="Screen Shot 2021-03-02 at 4 41 08 PM" src="https://user-images.githubusercontent.com/2240960/109712670-b5b7a180-7b76-11eb-996e-5ad2518d13bc.png">

List with the original order in the date range (see the refund has products now):
<img width="1663" alt="Screen Shot 2021-03-02 at 4 42 23 PM" src="https://user-images.githubusercontent.com/2240960/109712733-c536ea80-7b76-11eb-947f-8b9c62c0fd5a.png">

Reply:
Nice catch! I fixed it in the commit a523a4b